app/Providers/AuthServiceProvider.php

```diff
@@ -81,12 +74,7 @@ static function (): Auth0 {
     private function getLoginParameters(): array
     {
         $auth0LoginParameters = [];
-        if (config(KeycloakConfig::KEYCLOAK_LOGIN_ENABLED)) {
-            parse_str(config(KeycloakConfig::KEYCLOAK_LOGIN_PARAMETERS), $auth0LoginParameters);
-            return $auth0LoginParameters;
-        }
-
-        parse_str(config('auth0.login_parameters'), $auth0LoginParameters);
+        parse_str(config(KeycloakConfig::KEYCLOAK_LOGIN_PARAMETERS), $auth0LoginParameters);
```

Comment:
Maybe we no longer need a method?

Reply:
I think not, you need the parameters as an array not a string, & config() returns it as a string?

Reply:
I should have been more specific, I mean the method `getLoginParameters` ?

Reply:
Ah, now I get it.